src/maidsafe/vault/mpid_manager/mpid_manager.proto

```diff
@@ -1,4 +1,4 @@
-/*  Copyright 2012 MaidSafe.net limited
+/*  Copyright 2015 MaidSafe.net limited
```

Comment:
This is a new file just got created.
Shall the copyright date still to be 2012 or 2015 ?